pages/quantum.js

```diff
@@ -74,7 +74,7 @@ export default function Quantum() {
 
         <Breadcrumbs items={breadcrumbsMockItems} />
 
-        <div className="MarginTopBottom PaddingLeftRight">
+        <Container className="Margins PaddingLeftRight">
```

Comment:
Apparently using the container here is making the width of the example different from the others. Is this Container necessary here?
![image](https://github.com/catho/ssr-quantum-simulator/assets/105747728/780e039d-d697-4836-88d6-802a2d205e3b)